packages/migrations/composer.json

```diff
@@ -24,7 +24,7 @@
         "php": "^7.2",
         "doctrine/dbal": "^2.5",
         "doctrine/doctrine-migrations-bundle": "^1.3",
-        "doctrine/migrations": "^1.6.0",
+        "doctrine/migrations": "^1.8.1",
```

Comment:
Can you please mention this new requirement in upgrade notes?

Reply:
It is not necessary for me. Any previous version cannot even work from shopsys/migrations 7.0.0-beta-*